dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -71,5 +71,4 @@ def ensure_eyes_available
   return if @eyes
   @eyes = Applitools::Selenium::Eyes.new
   @eyes.api_key = CDO.applitools_eyes_api_key
-  @eyes.log_handler = Logger.new('../../log/eyes.log')
```

Comment:
Could you elaborate on this diff what it does, where the logs are found? how they were used?

Reply:
This log file was a raw transcript of the complete HTTP request and response for every API call we were making to the Applitools / Eyes web service. I enabled it and merged it, so that we could debug so issues we were having on our managed test server five years ago and forgot to turn it off. Maybe I should retain this logic but add a configuration setting or environment variable that allows us to toggle it on with a code change?